src/types.h

```diff
@@ -243,22 +243,21 @@ enum signature_type_t {
 //----------------------------------------------
 // etaglist_t / filepart
 //----------------------------------------------
-typedef std::vector<std::string> etaglist_t;
+typedef std::list<std::string> etaglist_t;
```

Comment:
Changed to `std::list` for pointer stability.

Reply:
LGTM.